ProcessList.c

```diff
@@ -647,6 +659,9 @@ void ProcessList_scan(ProcessList* this, bool pauseProcessUpdate) {
       }
    }
 
+   // Set UID column width based on max UID.
+   Process_setUidColumnWidth(maxUid);
```

Comment:
Do we need to call this every cycle or just when `maxUid` changed, e.g. after line 642?

Reply:
This needs to be called once the (final) `maxUid` has been found. Putting it on line 642 would result in it being called multiple times during the scan.